### DISCUSSION TITLE
Issue with the code

### DISCUSSION THREAD

**Discussion by ujjawaltyagii** [UniKonf/vibey#221] [General]
So for contributing in Vibey, I forked and done all the necessary set up on my local system and on running command `yarn dev` webpage was running on local host but when i tried opening the "events" in side menu bar i saw error and when i looked at code there were two errors in `./src/pages/events/index.tsx` (without even touching the code i am getting these errors in fresh cloned repo)

1st one was a missing closing div 

2nd one was that `sortEventsByDate` was using before declaring 

and on output I was getting this screen :

![image](https://github.com/UniKonf/vibey/assets/115401171/be3c3b83-3e1c-46ae-aba7-4aaf5115832d)

#### I fixed these minor errors and then went on to modify the according to the issue i was assigned with but then without even touching code I am getting this `toggle side-menu` different from the official website on my local host :
toggle menu on local host : (i didn't even touch the code)
![image](https://github.com/UniKonf/vibey/assets/115401171/02c5ef07-02ef-4478-bd2c-3e054018d42b)

on official website :
![image](https://github.com/UniKonf/vibey/assets/115401171/9961b51a-4158-40f1-b32b-a8c69e069ec9)

#### there is absolutely no comments in code to navigate a contributor through the code. And mostly important if these issues keep popping up then how are we gonna contribute smoothly

**Comment by ujjawaltyagii** [comment]
@J-Harsh  could you please clarify this ?!!!

**Comment by Deepu178** [comment]
@ujjawaltyagii  Thanks for pointing out the important issue. I have checked it out and fixed all the errors due to which deployment on Vercel was failing. It was the main reason you were getting toggle menu different on localhost than main website. We are working on these issues so that it will not occur in the future. We are also working on adding comments for better navigation to the new contributors. 
Thanks for giving the constructive feedback which will help Vibey to be more improved project.
